src/backend/postmaster/postmaster.c

```diff
@@ -1238,6 +1239,11 @@ PostmasterMain(int argc, char *argv[])
 	 */
 	process_shared_preload_libraries();
 
+	/*
+	 * Set CurrentMotionIPCLayer after interconnect.so is loaded.
+	 */
+	ResetCurrentMotionIPCLayer();
```

Comment:
Since we only use this function here for initialization, the function looks not a good name.

Reply:
fix in [commit](https://github.com/cloudberrydb/cloudberrydb/pull/646/commits/82b281fbea95edb8545b90632e8cf034fb6eb463)